test/InventoryItem.js

```diff
@@ -72,53 +75,45 @@ describe('InventoryItem', () => {
-    it('Should return 403 when an invalid token is supplied', done => {
-      chai
-        .request(app)
-        .post('/api/InventoryItem/addItem')
-        .send(ITEM_WITH_INVALID_TOKEN)
-        .then(function(res) {
-          expect(res).to.have.status(UNAUTHORIZED);
-          done();
-        })
-        .catch(err => {
-          throw err;
-        });
-    });
-    it('Should return 400 when the required fields aren\'t filled in', done => {
-      setTokenStatus(true);
-      chai
-        .request(app)
-        .post('/api/InventoryItem/addItem')
-        .send({ token, ...ITEM_WITHOUT_REQUIRED_FIELDS })
-        .then(function(res) {
-          expect(res).to.have.status(BAD_REQUEST);
-          done();
-        })
-        .catch(err => {
-          throw err;
-        });
+    it('Should return 403 when an invalid token is supplied', async () => {
+      const result = await test.sendPostRequestToken(
+        token, '/api/InventoryItem/addItem', ITEM_WITH_INVALID_TOKEN);
+      expect(result).to.have.status(UNAUTHORIZED);
     });
+    it('Should return 400 when the required fields aren\'t filled in',
+      async () => {
+        setTokenStatus(true);
+        const result = await test.sendPostRequestToken(
+          token, '/api/InventoryItem/addItem', ITEM_WITHOUT_REQUIRED_FIELDS);
+        expect(result).to.have.status(BAD_REQUEST);
+      });
     it('Should return statusCode 200 when all required fields are filled in',
-      done => {
+      async () => {
         setTokenStatus(true);
-        chai
-          .request(app)
-          .post('/api/InventoryItem/addItem')
-          .send({ token, ...VALID_NEW_ITEM })
-          .then(function(res) {
-            expect(res).to.have.status(OK);
-            done();
-          })
-          .catch(err => {
-            throw err;
-          });
+        const result = await test.sendPostRequestToken(
+          token, '/api/InventoryItem/addItem', VALID_NEW_ITEM);
+        expect(result).to.have.status(OK);
       });
   });
 
   describe('/GET getItems', () => {
     it('Should return an object of all items', done => {
       setTokenStatus(true);
+      // const result = await test.sendGetRequest(
+      //   '/api/InventoryItem/getItems');
+      // expect(result).to.have.status(OK);
+      // const getItemsResponse = result.body;
+      // getItemsResponse.should.be.a('array');
+      // expect(getItemsResponse).to.have.length(1);
+      // expect(getItemsResponse[0].name).to.equal(VALID_NEW_ITEM.name);
+      // expect(getItemsResponse[0].price).to.equal(VALID_NEW_ITEM.price);
+      // expect(getItemsResponse[0].stock).to.equal(VALID_NEW_ITEM.stock);
+      // expect(getItemsResponse[0].category).to
+      //   .equal(VALID_NEW_ITEM.category);
+      // expect(getItemsResponse[0].picture).to.equal(VALID_NEW_ITEM.picture);
+      // expect(getItemsResponse[0].description).to.equal(
+      //   VALID_NEW_ITEM.description);
+      // itemName = getItemsResponse[0].name;
```

Comment:
In the routes, this is a POST call, not a get call, so I was a bit confused.

Reply:
Feel free to change it to `GET`